core/src/main/scala/com/salesforce/op/stages/impl/preparators/SanityChecker.scala

```diff
@@ -274,18 +274,6 @@ class SanityChecker(uid: String = UID[SanityChecker])
       .toMap[String, Double]
     val numCorrIndices = corrIndices.length
 
-    // build map from indicator group name to any null indicator it may have
-    val nullGroups = for {
-      col <- metaCols
-      if col.isNullIndicator
-        grouping <- col.grouping
-      } yield (grouping, (col, col.index))
-
-    nullGroups.groupBy(_._1).foreach {
```

Comment:
Do you think we should take out the whole check for multiple null indicators right now?

Afaik, we still have an issue when a single parent feature is vectorized with two different unary stages (eg. if you were to vectorize a real with both the normal `RealVectorizer` and also bucketize). SanityChecker will still group these into a single contingency matrix since their grouping will just be the parent feature name, so I think we should leave this check in until we fix that issue.

Reply:
Do you really think it is better to fail the whole flow when that happens? I would be better to automatically drop one than fail the flow...

Reply:
Or even have some incorrect statistics (in my opinion)

Reply:
Can we at least keep it in as a warning we log so that we can monitor when it's happening?

Reply:
@Jauntbox I fixed it so that it will remove duplicates from the categorical calculations